i18n/en.toml

```diff
@@ -198,6 +198,10 @@ other = "Subscribe"
 [synopsis_heading]
 other = "Synopsis"
 
+
+[thirdparty_message]
+other = """This section links to third party projects that provide functionality required by Kubernetes. The Kubernetes project authors aren't responsible for these projects. This page follows <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF website guidelines</a> by listing projects alphabetically. To add a project to this list, read the <a href="/contribute/style/content-guide/#third-party-content">content guide</a> before submitting a change."""
```

Comment:
The link to the content guide is not correct.
When I mentioned splitting up the thirdparty_message string, I envisioned you could include most of the
text but create different variables for the "content-guide" string and the "cncf guideline" string. The shortcode
would put the pieces together and offer a way to mix and match the message. This may be more difficult to localize.

Reply:
I'd put Markdown in the message rather than HTML - most contributors and localization teams can work with both, but Markdown is our norm.

Reply:
@sftim All message that use links use HTML (for the moment). I recommend rather than changing them to markdown in this PR, we open a seperate issue and follow-up after – Any shortcode or partial that uses translated strings will need to be changed to use the `markdownify` pipe from (probably) the `safeHTML` pipe.

Reply:
👍